src/app/canary/canary.component.html

```diff
@@ -0,0 +1,6 @@
+<div id="canary-holder">
+    <div tabindex="0" [attr.aria-label]="getStr('Using demo tenant')">{{getStr('using canary')}}</div>
```

Comment:
We need to style this div to match the padding-right of #disable-canary

Reply:
Also the `aria-label` does not match the string being shown to the user.
It's value is `Using demo tenant` instead of `using canary`